assets/js/voronoi.js

```diff
@@ -26,6 +32,11 @@ const dropArea = document.getElementById("dropzone");
 dropArea.addEventListener("drop", handleDrop, false);
 dropArea.addEventListener("input", handleChange, false);
 
+numPointsSlider.oninput = function () {
```

Comment:
I don't like this style of handling events. Better use `addEventListener` and arrow function

Reply:
Updated!